labs/computing-experiment-metrics/README.md

```diff
@@ -1,11 +1,12 @@
-# Computing Experiment Datasets #1: Experiment Subjects
+# Optimizely Labs Template Notebook
```

Comment:
Can you specify `Optimizely Labs Template for Jupyter Notebooks` in the title? Otherwise it seems like this is the generic template for all Optimizely Labs.

Reply:
Also, if we truly want to make a template for Jupyter notebooks, can you move the template pieces into the `templates` folder rather than the `labs` folder?

That way we don't get it confused with an actual lab and we can also reference it more easily in the `CONTRIBUTING` instructions

Reply:
I removed the "template" language in `README.md` and replaced it with lab-specific language.  Thank you for catching this!